etc/helm/pachyderm/templates/pachd/deployment-id-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- /*
```

Comment:
Do we need this under enterprise server only deployments as well? (`templates/enterprise-server`)